src/class.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+class BaseTree {
+  constructor() {
+    this.fruit = 0;
+  }
+}
+
+class Tree extends BaseTree {
+  constructor() {
+    super();
+  }
+  makeFruits(){
+    ++this.fruit;
+  }
+}
+
+function main(numPpl=15, numFruits=1) {
+  var plants = [];
+  var week = 1;
+
+  calculate(numPpl, numFruits);
+  function calculate(numPpl, numFruits){
+    if (week ===1) {
+      for (var ii = 0; ii < numFruits; ii++) {
+        plants.push(new Tree);
```

Comment:
I'm surprised the code still run, did not throw me an error so that I could know I missed out a pair of () @@